### DISCUSSION TITLE
PD24 starts service during the installation conflict with "-BlockExecution".

### DISCUSSION THREAD

**Discussion by Hardexit** [PSAppDeployToolkit/PSAppDeployToolkit#815] [Q&A] [answered]
Hello all,

I want to install pdf24 via PSAppDeployToolkit, but this runs into errors because the process is started as a service during installation.

`Show-InstallationWelcome -CloseApps 'pdf24' -BlockExecution -AllowDefer -DeferTimes 3 -DeferDays 3`

Is there a workaround for this or do I need to remove `-BlockExecution` from the parameters?

**Comment by That-Annoying-Guy** [comment]
Stop the service.

Block execution is for apps that users will try to launch

PS: maybe look here for help: https://discourse.psappdeploytoolkit.com/c/thetoolkit/general-discussion/11

**Reply by Hardexit** [reply]
How do I stop the service that is started during the installation of the msi?

**Reply by mjr4077au** [reply, marked as answer]
@Hardexit I'm not sure if this is still an issue, but your service auto-starting can likely be changed by applying an MST to alter the MSI install process to not do this. How that might be possible is probably beyond the discussion we can have here.

Have a look at the very fine Master Packager: https://www.masterpackager.com/master-packager. It's available for free and it should allow you to see what changes need to be made to the MSI to remove this CustomAction and save it as an MST.
